snap/README.md

```diff
@@ -89,3 +89,31 @@ or
 sudo snap install edgex-app-service-configurable edgex-app-service-configurable_mqtt
 ```
 **Note** – you must ensure that any configuration values that might cause conflict between the multiple instances (e.g. port, log file path, …) must be modified before enabling the snap’s service.
+
+### Secret Store Usage
+Some profile configuration.toml files specify configuration which requires secret-store support, however this snap doesn't fully support secure secrets without manual intervention (see below). The snap can also be configured to use insecure secrets as can be done via docker-compose by setting the option ```security-secret-store=false```. Ex.
+
+```
+sudo snap set edgex-app-service-configurable security-secret-store=false
+```
+
+### Manually install CA root certificate & vault token
+Here's an example of how to use a configuration/profile which includes secrets:
+
+```
+$ sudo snap install edgex-app-service-configurable --channel=latest/candidate/alpha
+$ sudo snap set profile=mqtt
+$ cd /var/snap/edgex-app-service-configurable/current/config/res/mqtt-profile
+$ sudo cp /var/snap/edgexfoundry/secrets/ca/ca.pem .
+$ sudo cp /var/snap/edgexfoundry/secrets/edgex-application-service/secrets-token.json .
+```
+
+Next the profile's configuration.toml file needs to be updated to reference the paths in ```$SNAP_DATA``` of the copied files. The two config fields that need updated are ```  RootCaCertPath``` and ```TokenFile```. The following shows how this can be done via the sed command-line tool, however the file can also be easily updated via your favorite editor.
+
```

Comment:
So the configuration.toml file in /var/snap/edgex-app-service-configurable/current/config/res/mqtt-export ?